src/index.html

```diff
@@ -7,8 +7,43 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Raleway:ital,wght@0,100;0,400;0,600;1,900&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active ">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover">Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

Reply:
Pay attention to formatting
<img width="1013" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/f1aaaba4-7d4b-451e-97ef-c64e5fb30d51">
Fix similar problem in all places